www/js/app.js

```diff
@@ -837,8 +837,14 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                         return false;
                     });
                 }
-                else if (url.slice(0, 1) === "#") {
-                    // It's an anchor link : do nothing
+                else if (util.startsWith($(this).attr("href"), "#")) {
```

Comment:
Hi @mossroy , I think this could be written much more simply as:
`else if (this.hash) {`
It matches if the URL is an anchor, and doesn't need jQuery. And doesn't need a utility written for it.
Yours works fine, but this could be easier?

Reply:
The problem is that we want to match only local anchors, but don't want to match anchors that would be in another URL.
I suppose (this.hash) is true for any anchor